lib/ClangImporter/ImportMacro.cpp

```diff
@@ -381,7 +388,9 @@ static ValueDecl *importMacro(ClangImporter::Implementation &impl,
 
     // If it's a literal token, we might be able to translate the literal.
     if (tok.isLiteral()) {
-      return importLiteral(impl, DC, macro, name, tok, ClangN, castType);
+      return importLiteral(
+          impl, DC, macro, name, tok, ClangN, castType,
+          isOriginalMacroFunctionLike || macro->isFunctionLike());
```

Comment:
Hm, okay. I'm suspicious of cases like `#define BAR FOO()` but we can wait until that actually happens to fix this.